packages/lesswrong/components/posts/PostsPage/PostsPage.jsx

```diff
@@ -296,84 +296,86 @@ class PostsPage extends Component {
 
       const commentId = query.commentId || params.commentId
       return (
-          <AnalyticsContext pageContext="postsPage">
+          <AnalyticsContext pageContext="postsPage" postId={post._id}>
             <div className={classNames(classes.root, {[classes.tocActivated]: !!sectionData})}>
               <HeadTags url={Posts.getPageUrl(post, true)} canonicalUrl={post.canonicalSource} title={post.title} description={description}/>
               {/* Header/Title */}
-              <div className={classes.title}>
-                <div className={classes.post}>
-                  {commentId && <CommentPermalink documentId={commentId} post={post}/>}
-                  {post.groupId && <PostsGroupDetails post={post} documentId={post.groupId} />}
-                  <AnalyticsContext pageSectionContext="topSequenceNavigation">
-                    <PostsTopSequencesNav post={post} sequenceId={sequenceId} />
-                  </AnalyticsContext>
-                  <div className={classNames(classes.header, {[classes.eventHeader]:post.isEvent})}>
-                    <div className={classes.headerLeft}>
-                      <PostsPageTitle post={post} />
-                      <div className={classes.secondaryInfo}>
-                        <span className={classes.authors}>
-                          <PostsAuthors post={post}/>
-                        </span>
-                        <span className={classes.postType}>
-                          <ContentType type={contentType}/>
-                        </span>
-                        { post.feed && post.feed.user &&
-                          <Tooltip title={`Crossposted from ${feedLinkDescription}`}>
-                            <a href={feedLink} className={classes.feedName}>
-                              {post.feed.nickname}
-                            </a>
-                          </Tooltip>
-                        }
-                        {!!wordCount && !post.isEvent &&  <Tooltip title={`${wordCount} words`}>
-                            <span className={classes.wordCount}>{parseInt(wordCount/300) || 1 } min read</span>
-                        </Tooltip>}
-                        {!post.isEvent && <span className={classes.date}>
-                          <PostsPageDate post={post} hasMajorRevision={hasMajorRevision} />
-                        </span>}
-                        {post.types && post.types.length > 0 && <Components.GroupLinks document={post} />}
-                        <a className={classes.commentsLink} href={"#comments"}>{ Posts.getCommentCountStr(post)}</a>
-                        <span className={classes.actions}>
-                          <AnalyticsContext pageElementContext="tripleDotMenu">
-                            <PostsPageActions post={post} />
-                          </AnalyticsContext>
-                        </span>
+              <AnalyticsContext pageSectionContext="postHeader">
+                <div className={classes.title}>
```

Comment:
This diff would be a lot more readable if these were merged together, either like

```
<AnalyticsContext pageSectionContext="postHeader"><div className={classes.title}>
```
or like
```
<AnalyticsContext pageSectionContext="postHeader">
<div className={classes.title}>
```
(with no extra indent level).